### ISSUE TITLE
Display of Connection types

### ISSUE THREAD

**Issue by tcstewar** [nengo/nengo_gui#598]
It'd be nice to have a few different Connection types displayed in some way.  For example:

- normal NEF connections
- inhibitory connections
- excitatory connections (?)
- learning connections
- modulatory connections

These could be different end-markers on the lines, or maybe different colours.

**Comment by astoeckel** [comment]
Since this is a feature I wanted to have myself for quite a long time here, I'm going to assign myself, as discussed with @tcstewar and @jgosmann today. Plan is to merge such "small" UI changes before the refactoring to try them out, being perfectly aware that they will be thrown away once the refactoring is done.

**Comment by astoeckel** [comment]
I've implemented a first version of this (see pull request #XXX). So far I've got inhibitory, normal, and connections attached to a learning rule (I call those modulatory in the code, may be wrong). @tcstewar  can you elaborate on the difference between a learning connection and a modulatory connection?

**Comment by tcstewar** [comment]
> can you elaborate on the difference between a learning connection and a modulatory connection?

The "learning connection" is the one whose weights are changing (i.e. the one with the learning rule), and the "modulatory connection" is the one providing the signal that changes the weights (i.e. the one connected to the learning rule).  :)

**Comment by astoeckel** [comment]
Ah, then I've used the terms correctly, thanks for the clarification. Any idea how the "learning connection" may be displayed? I figure that in my current implementation it should be fairly obvious that a connection is a "learning connection" because it has the modulatory connection attached to it.

**Comment by krisfrenette** [comment]
Is this completed then? If not can someone please add what is left to complete this story.

**Comment by tcstewar** [comment]
I think we've got the important ones added.  There's more that could be done, but I don't think there's a strong consensus about what extra indications would be good without adding unnecessary clutter.
